packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -121,6 +121,33 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       buckets: [0.1, 1, 2, 4, 10],
       labelNames: ["source"],
     }),
+    blockProductionTimeSteps: register.histogram<"step" | "source">({
```

Comment:
from now on we don't want to track more than one label per metric due to its performance issue, see https://github.com/ChainSafe/js-libp2p-gossipsub/pull/433